openstack/limes-campfire/values.yaml

```
@@ -5,3 +5,12 @@ campfire:
   image_tag: null # defined in values file
   cpu_request: 20m
   memory_limit: 64Mi
+
+owner-info:
+  support-group: containers
+  service: limes-campfire
```

```suggestion
  service: limes
```
Otherwise pod alerts will not show up in the limes alert channel.